### ISSUE TITLE
can i use my own pics to get the result

### ISSUE THREAD

**Issue by BumbleBaozi** [thanif/Camera-Calibration-through-Camera-Projection-Loss#6]
Hello, thank you for sharing. Secondly, I would like to ask whether I can use the checkerboard image taken by myself as input to get internal and external parameters. I haven't been able to run through your code yet. May I ask how simple use, can achieve the above results. Looking forward to your reply! Thanks!

**Comment by thanif** [comment]
Yes, you can use your own pics to get the result as follows:

Left_images = []
Right_images = []

Read left and right image:

l_im = cv2.imread(data_path+folder+"/"+fname, 0)
r_im = cv2.imread(data_path+folder+"/"+'RightRGB_'+fname.split("_")[1], 0)

Left_images.append(l_im)
Right_images.append(r_im)

Save images as npy files:

np.save(path+"li.npy",Left_images)
np.save(path+"ri.npy",Right_images)

Load Data:

Left_images = np.load(data_path+"li.npy")
Right_images = np.load(data_path+"ri.npy")

Load the weights:

model.load_weights('./new_logs/20221209-143908/model_multi_class/Best/weights_07_486.93.h5')

Test the model:

output = model.predict(
    x=[Left_images, Right_images],
    batch_size=16,
    verbose=1
)

**Comment by BumbleBaozi** [comment]
Thanks for u replay. I am trying to get the rsesult. Unfortunately, what about  these as follows?
Fx = np.load("Fx.npy")
Fy = np.load("Fy.npy")
U0 = np.load("U0.npy")
V0 = np.load("V0.npy")
Baseline = np.load("Baseline.npy")
Disparity = np.load("Disparity.npy")
Tx = np.load("Tx.npy")
Ty = np.load("Ty.npy")
Tz = np.load("Tz.npy")
Pitch = np.load("Pitch.npy")
X = np.load("X.npy")
Y = np.load("Y.npy")
Z = np.load("Z.npy")

How is the initialization of these parameter values defined?  How do I generate the '.npy' file above? 
Looking forward to your reply! Thanks!

**Comment by BumbleBaozi** [comment]
So sorry to bother you again. I recently studied your code and found that the 112*112 size image can get the calibration result. But because my experiment needs to be on a 3648*5472 size picture, I saw that your code involves a Dense fully connected layer. After consulting the information, it seems that an image with the same size as the weight file needs to be input. I don't know if there is a solution, thank you, looking forward to your reply.

**Comment by BumbleBaozi** [comment]
So sorry to bother you again. I recently studied your code and found that the 112x112 size image can get the calibration result. But because my experiment needs to be on a 3648x5472 size picture, I saw that your code involves a Dense fully connected layer. After consulting the information, it seems that an image with the same size as the weight file needs to be input. I don't know if there is a solution, thank you, looking forward to your reply.
